From d674b41610451381eb21d7a6828a2408756f9b42 Mon Sep 17 00:00:00 2001
From: Marcel Ziswiler <marcel@ziswiler.com>
Date: Wed, 25 Jun 2014 22:10:01 +0200
Subject: [PATCH 34/45] ARM: tegra: Migrate Apalis T30 PCIe power supply scheme

This migration is required for continued PCIe operation after commit
d3c7e24b84fc "PCI: tegra: Implement accurate power supply scheme".

Signed-off-by: Marcel Ziswiler <marcel@ziswiler.com>
[swarren: added commit subject and shortened hash]
Signed-off-by: Stephen Warren <swarren@nvidia.com>
---
 arch/arm/boot/dts/tegra30-apalis.dtsi | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/tegra30-apalis.dtsi b/arch/arm/boot/dts/tegra30-apalis.dtsi
index d04efcf..8adaa78 100644
--- a/arch/arm/boot/dts/tegra30-apalis.dtsi
+++ b/arch/arm/boot/dts/tegra30-apalis.dtsi
@@ -9,9 +9,14 @@
 	compatible = "toradex,apalis_t30", "nvidia,tegra30";
 
 	pcie-controller@00003000 {
-		pex-clk-supply = <&sys_3v3_reg>;
-		vdd-supply = <&vdd2_reg>;
-		avdd-supply = <&ldo6_reg>;
+		avdd-pexa-supply = <&vdd2_reg>;
+		vdd-pexa-supply = <&vdd2_reg>;
+		avdd-pexb-supply = <&vdd2_reg>;
+		vdd-pexb-supply = <&vdd2_reg>;
+		avdd-pex-pll-supply = <&vdd2_reg>;
+		avdd-plle-supply = <&ldo6_reg>;
+		vddio-pex-ctl-supply = <&sys_3v3_reg>;
+		hvdd-pex-supply = <&sys_3v3_reg>;
 
 		pci@1,0 {
 			nvidia,num-lanes = <4>;
-- 
2.0.1

